packages/calling/src/CallingClient/registration/register.ts

```diff
@@ -169,7 +172,7 @@ export class Registration implements IRegistration {
     const deviceInfo = {
       userId: this.userId,
       clientDeviceUri: this.webex.internal.device.url,
-      serviceData: this.serviceData,
+      serviceData: this.jwe ? {...this.serviceData, jwe: this.jwe} : this.serviceData,
```

Comment:
Would this change not require test file update?

Reply:
This has been tested in the line.test.ts file.